src/chainstate/stacks/boot/costs-2.clar

```diff
@@ -1 +1,567 @@
-;; the .costs-2 contract
\ No newline at end of file
+;; the .costs-2 contract
```

Comment:
Can you also put this into the SIP 012 text?  Thanks!

Reply:
working on it!